tests/conftest.py

```diff
@@ -248,8 +233,13 @@ def rl_ray_version(request):
 
 
 @pytest.fixture(scope="module")
-def chainer_full_version(request):
-    return request.config.getoption("--chainer-full-version")
+def chainer_full_version():
```

Comment:
Out of scope, but are we still supporting Chainer?

Reply:
There's been no public announcement to say we're not supporting it.